guide/yaml/test/test-entities.md

```diff
@@ -89,6 +93,10 @@ The `TestEffector` entity invokes the specified effector on a target entity. If
 - `params` - parameters to pass to the effector, these will depend on the entity and effector being tested. The example above shows the `url` and `targetName` parameters being passed to Tomcats `deploy` effector.
 - `assert` - assertion to perform on the returned result. See section on assertions below.
 
+<div class="alert alert-info">
```

Comment:
this would be a good candidate for a partial/include following the pattern of `_includes/feature-item.html` as used in `website//learnmore/features/*.md` -- you could just write

    {% include tip-testcase.html title="TestEffector" %}

also curious why you're using html rather than md syntax

but both comments are pedantic, feel free to ignore

merging